### PR TITLE
feat(core): nami migration customization step

### PR DESCRIPTION
**Pull request by renanvalentin**
# Checklist

- [x] JIRA - [LW-10497](https://input-output.atlassian.net/browse/LW-10497)
- [x] Screenshots added.

---

⚠️ Merge https://github.com/input-output-hk/lace-with-nami-mode/pull/5 first 

## Screenshots

Screenshot 2024-05-21 at 14 33 16
Screenshot 2024-05-21 at 14 33 20
Screenshot 2024-05-21 at 14 33 31
Screenshot 2024-05-21 at 14 33 37

### CODE REVIEW

File: packages/core/src/ui/components/NamiMigration/Customize/Customize.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { sx, Flex, Text, Box, Button, ToggleButtonGroup } from '@lace/ui';
```
Comment (lucas-barros) [inline packages/core/src/ui/components/NamiMigration/Customize/Customize.tsx:3]: > 'sx' is defined but never used

Comment (renanvalentin) [reply packages/core/src/ui/components/NamiMigration/Customize/Customize.tsx:3]: Shouldn't git commit or git push catch these issues?

**Review by DominikGuzei — APPROVED** [review]
I can see there is a merge conflict, otherwise great work 👍

File: packages/core/rollup.config.js [hunk 2]
```diff
@@ -23,9 +24,16 @@ export default (args) => {
     plugins: [
       ...baseConfig.plugins,
       copy({
-        targets: [{ src: 'src/ui/lib/translations/en.json', dest: 'dist/translations/' }]
+        targets: [
+          { src: 'src/ui/lib/translations/en.json', dest: 'dist/translations/' }
+          // { src: 'src/ui/components/NamiMigration/Customize/*.mp4', dest: 'dist/' }
```
Comment (lucas-barros) [inline packages/core/rollup.config.js:29]: left over comment.

File: packages/core/src/ui/components/NamiMigration/Customize/Customize.tsx (hunk 1 above)
Comment (lucas-barros) [reply packages/core/src/ui/components/NamiMigration/Customize/Customize.tsx]: I'm not sure, but it should.

File: packages/core/rollup.config.js (hunk 2 above)
Comment (lucas-barros) [reply packages/core/rollup.config.js:29]: @renanvalentin.

Comment (renanvalentin) [reply packages/core/rollup.config.js]: @lucas-barros fixed

**Review by lucas-barros — APPROVED** [review]

**Pull request merged**